components/ExploreTab.vue

```diff
@@ -0,0 +1,116 @@
+<script setup lang="ts"></script>
+<template>
+  <div class="cover-div">
+    <h1>Upcoming Book Fair</h1>
+    <img src="~/assets/images/books.jpeg" />
+    <button>Book</button>
+  </div>
+  <div class="popular-div">
+    <h2>Popular Now</h2>
+    <p>See All</p>
+  </div>
+  <div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel">
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <div>
+          <img src="~/assets/images/tools.jpeg" class="d-block w-100" alt="a" />
+          <div class="img-des">
+            <h3>Zero to One</h3>
+            <p>Blake Masters & Peters</p>
+          </div>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <div>
+          <img src="~/assets/images/elon.jpeg" class="d-block w-100" alt="elon" />
+          <div class="img-des">
+            <h3>Elon Musk</h3>
+            <p>Ashlee Vance</p>
+          </div>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <div>
+          <img src="~/assets/images/jen.jpeg" class="d-block w-100" alt="jen" />
+          <div class="img-des">
+            <h3>You are a badass</h3>
+            <p>Jen sincero</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="prev">
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="visually-hidden">Previous</span>
+    </button>
+    <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="next">
+      <span class="carousel-control-next-icon" aria-hidden="true"></span>
+      <span class="visually-hidden">Next</span>
+    </button>
+  </div>
+</template>
+<style lang="scss" scoped>
+@import "assets/sass/main.scss";
```

Comment:
Why have you imported the `main.scss` file here? It's already declared in the `nuxt.config`.

Reply:
if i don't import `main.scss` then it gives this error
<img width="876" alt="Screenshot 2022-11-17 at 2 18 44 PM" src="https://user-images.githubusercontent.com/79077765/202393484-88940723-d6fd-484f-9043-b08ffe9b5f42.png">